css/main.css

```diff
@@ -0,0 +1,95 @@
+.side-bar{
+    font-family: 'Open Sans', sans-serif;
+    flex: 0 0 200px;
+    margin-right: 1rem;
+    background-color: var(--bg-color);
+}
+
+.filter-heading{
+    padding: 10px; 
+}
+
+.filter-heading h5{
+    color: var(--default-color);
+    font-weight: 600;
+    font-size: 1.4rem;
+    background-color: var(--default-color);
+    color: var(--off-white);
+    padding: 5px;
+    
+}
+
+.filter-heading h4{
+    color: var(--default-color);
+    font-weight: 600;
+    font-size: 1.2rem;
+    border-bottom: 2px solid var(--default-color);
+}
+
+label{
+    border: none;
+}
+
+.filter-list{
+    margin-left: 10px;
+}
+
+.slider {
+    width: 100%;
+  }
+
+  .minmax-value{
+      display: flex;
+      justify-content: space-between;
+  }
+
+.doc-content{
+    min-width: 0;
+    margin: 0px 2rem;
+    max-width: 90%;
+    margin-left: 0%;
+    margin-right: 0%;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 5px;
+}
+
+.card-vertical {
+    max-width: 300px;
+    width: 100%;
```

Comment:
This can be simplified to `width: 300px`

Reply:
I actually did not get it, max width is 300 px and less than that will be 100%. That is what I wanted